ui/pages/confirmations/confirmation/alerts/setAlerts.ts

```diff
@@ -0,0 +1,26 @@
+import { useEffect, useMemo } from 'react';
+import { useDispatch } from 'react-redux';
+import {
+  Alert,
+  clearAlerts,
+  updateAlerts,
+} from '../../../../ducks/confirm-alerts/confirm-alerts';
+import { useI18nContext } from '../../../../hooks/useI18nContext';
+
+const setAlerts = (alertOwnerId: string) => {
+  const dispatch = useDispatch();
+  const t = useI18nContext();
+  const alerts: Alert[] = useMemo(() => [], [t]);
```

Comment:
Minor, do we usually both with the `t` dependency since it's a context set at the root of the UI so doesn't change?

Reply:
fixed